test/cli/dbt-init/ok-interactive/interaction.txt

```diff
@@ -0,0 +1,11 @@
+< Please enter Keboola Storage API host, eg. "connection.keboola.com".
+> %%TEST_KBC_STORAGE_API_HOST%%
+
+< Please enter Keboola Storage API token. The value will be hidden.
+> %%TEST_KBC_STORAGE_API_TOKEN%%
+
+< Please enter target name.
+> target1
+
+< Enter a name for a workspace to create:
+> dbt_source
```

Comment:
@michaljurecko Prosím tě já to s těma interaktivníma testama nějak neumím. 😓 

![image](https://user-images.githubusercontent.com/215660/196739264-fdccda4d-af1c-45e7-900b-e7e0f5eb1a4d.png)

Output ze CLI vypadá teda takto: 
![CleanShot 2022-10-19 at 17 46 00](https://user-images.githubusercontent.com/215660/196740561-a7d2a0ba-3a97-4c4e-8672-f24d9b56f8c5.jpg)

To `Enter a name for a workspace to create` je až za otazníkem, takto možná nemá být tady stejně jako třeba `API host`? Ale nefunguje to ani bez něj tak nevím:

![CleanShot 2022-10-19 at 17 47 50](https://user-images.githubusercontent.com/215660/196741042-ff7d24a7-d94e-48ac-b64c-bb215d16a07e.jpg)

Reply:
:eyes:

Reply:
BTW, nechcel by si pouzivat na kazdy PR samostatnu branch ? :slightly_smiling_face: a v nej mat to `*-KAC-123-*` ?
To som sa chcel uz viac krat spytat, ale vzdy zabudnem.

Reply:
To už jsem zkoušel, ale pak jsem musel pořád dohledávat na jakým čísle že to vlastně pracuju, když jsem potřeboval přepnout. 🙂 Ale jestli to vás nějak plete, tak jasně.

Reply:
Ako mne je to v principe jedno, ale vraj mame taky standard v Kebool-e.
A celkom sa hodi najst si to z Jiry ako link na Github.

Tu je fix problemu, moja chyba: https://github.com/keboola/keboola-as-code/pull/911

Reply:
No standard je prolinkování s Jirou, to samozřejmě dělám. O pojmenování branchí nevím. Dřív jsem psal id issue do popisku PR 

<img width="723" alt="image" src="https://user-images.githubusercontent.com/215660/196870122-932fedfd-902f-4f10-b752-14178d75286f.png">

A potom jsem (od tebe) zjistil že stačí když se link na issue dá do description PR. A to má navíc výhodu v tom, že se dá rovnou prokliknout. Tady je vidět, že to prolinkovaný je:

![CleanShot 2022-10-20 at 08 10 31@2x](https://user-images.githubusercontent.com/215660/196869667-32471a73-967b-440e-b115-548ce8c24814.jpg)

Ale jak říkám, pokud ti to nějak komplikuje život, tak to zvládnu. 😄

Reply:
> Tu je fix problemu, moja chyba: https://github.com/keboola/keboola-as-code/pull/911

Kouknu, díky!

Reply:
Ok, takze ak je v PR link na Jiru, tak sa to tiez prelinkuje?
Alebo to pridavas rucne?
Mne len Vojta na onboardingu hovoril, ze musim "vytvarat PR s issue tagom v branch", ak to ide teda aj inak, tak len o tom neviem :slightly_smiling_face:

Reply:
jj. já myslel že tam ten link do PR dáváš kvůli tomu. 😄 Tak jsem to zkusil a zjistil že se to zvládne prolinkovat (automaticky, ručně to nedělám). Popsaný jak to referencování funguje je tady: https://support.atlassian.com/jira-software-cloud/docs/reference-issues-in-your-development-work/ (ale nic moc navíc tam asi už není)

Reply:
ok. ja som zil v tom, ze ked tam nieke to ID v branch name, tak nejaka integracia nefunguje. Dik za vysvetlenie :+1: